global/settings/types/preferencekeys.h

```diff
@@ -64,6 +64,7 @@
 #define PREF_IMPORT_MUSICXML_IMPORTLAYOUT                   "import/musicXML/importLayout"
 #define PREF_IMPORT_OVERTURE_CHARSET                        "import/overture/charset"
 #define PREF_IMPORT_STYLE_STYLEFILE                         "import/style/styleFile"
+#define PREF_IMPORT_RESET_OLD_SCORES                        "import/style/resetOldScores"
```

Comment:
Is `style` really the best subcategory to use here?

Maybe something like `import/compatibility/resetElementPositions` instead?

(And change the macro to `PREF_IMPORT_COMPATIBILITY_ RESET_ELEMENT_POSITIONS`.)

Reply:
That "styles" was a copy/past thing. Maybe just "import/resetOldScores"

Reply:
There's at least one other compatibility fix that I was potentially thinking of adding in the future. So it might be useful to have “compatibility” as a subcategory. And “reset element positions” is more specific than “reset old scores”.